test-app-sitebricks.yaml

```diff
@@ -78,41 +78,43 @@ paths:
           description: "return balance of user successfully"
         400:
           description: "invalid user id"
-  /accounts/{userId}/deposit/{value}:
+  /accounts/{userId}/deposit:
     post:
       summary: "deposit amount to user account"
       operationId: "deposit"
       parameters:
+        - in: "body"
+          name: "amount"
+          description: "amount to deposit"
+          required: true
+          schema:
+            type: "integer"
```

Comment:
in my implementations is Integer because I think about that balance as token balance, which is my mistake, balance field will be changed to tokens or tokenBalance